### PR TITLE
Magazine Cover, A11Y Form and Cyclomatic Complexity Homework

### PR DESCRIPTION
**Pull request by JuanFelipeChaparro**

### CODE REVIEW

File: bogota-abril-2019/css-magazine-cover/JuanChaparro/index.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    <body>
+        <div id="container">
+            <div id="main-container">
+                <header>
+                    <h3>April 12, 2010</h3>
+                    <h2><span>Environment Special:</span> The Perils of Plastic</h2>
+                    <h1>TIME</h1>
+                </header>
+                <footer>
+                    <h2>Inside</h2>
```
Comment (gabrielmartinezglobant) [inline bogota-abril-2019/css-magazine-cover/JuanChaparro/index.html]: Incorrect use of semantic. If the heach one of the h2 is the same context should only be one

File: bogota-abril-2019/css-magazine-cover/JuanChaparro/styles.css [hunk 2]
```diff
@@ -0,0 +1,68 @@
+#container {
+    border: 50px solid #ED1C24;
+    width: 1000px;  
+    margin: auto;
+}
+
+#main-container {
```
Comment (gabrielmartinezglobant) [inline bogota-abril-2019/css-magazine-cover/JuanChaparro/styles.css]: Avoid use of id selector. Usually is a code smell

File: bogota-abril-2019/cyclomatic-complexity/JuanChaparro/unluckyDays.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+function unluckyDays(year) {
+    var countUnlucky = 0;
+    var mDate = new Date(year, 0, 1);
```
Comment (gabrielmartinezglobant) [inline bogota-abril-2019/cyclomatic-complexity/JuanChaparro/unluckyDays.js]: what does mDate represents?

Comment (JuanFelipeChaparro) [reply bogota-abril-2019/cyclomatic-complexity/JuanChaparro/unluckyDays.js]: I usually use m to represent 'My', so in this case is 'My Date'.
I changed it to dummyDate to be more specific.

File: bogota-abril-2019/css-magazine-cover/JuanChaparro/index.html (hunk 1 above)
Comment (JuanFelipeChaparro) [reply bogota-abril-2019/css-magazine-cover/JuanChaparro/index.html]: Done.

File: bogota-abril-2019/css-magazine-cover/JuanChaparro/styles.css (hunk 2 above)
Comment (JuanFelipeChaparro) [reply bogota-abril-2019/css-magazine-cover/JuanChaparro/styles.css]: I corrected it, i have a question.
When should I use id selector?

**Pull request merged**